Source/sound.cpp

```diff
@@ -43,7 +43,7 @@ void LoadMusic(HANDLE handle)
 	SDL_RWops *musicRw = SFileRw_FromStormHandle(handle);
 #else
 	int bytestoread = SFileGetFileSize(handle, 0);
-	musicBuffer = (char *)DiabloAllocPtr(bytestoread);
+	musicBuffer = new char[bytestoread];

```

Comment:
if you need something todo then feel free to apply `std::byte` to things like we talked about :)

Reply:
`std::byte` isn't available on GCC 5/6 :(

Reply:
`#define std::byte std::uint8_t` :shrug: 

or we could `typedef` our own `byte` type based on compiler ability?.

The current situation is a bit of a confusing mess with `uint8_t`, `BYTE`, and `char` being used in various part of the code :( Don't feel any pressure though, if you don't to anything I will at some point :)